### PR TITLE
Show column breakpoints when line breakpoint is set, remove Alt/Option key dependency

### PR DESCRIPTION
**Pull request by audreyfang**
### Summary of Changes

* Pressing Option/Alt only shows column breakpoints on lines that have a breakpoint already set.

### Screenshots/Videos (OPTIONAL)
column-breakpoints

### CODE REVIEW

**Comment by darkwing** [comment]
Hello @audreyfang !  Could you update this with next steps?

File: src/components/Editor/CallSites.js [hunk 1]
```diff
@@ -142,17 +142,34 @@ class CallSites extends Component {
       });
     }
   }
+  filterCallSites() {
```
Comment (darkwing) [inline src/components/Editor/CallSites.js]: I'd love a more specific name on this function; i.e. filtered by what?  Additionally, a comment above the function as to why the given filter is there would be nice.

**Comment by darkwing** [comment]
After a bit of quick testing:

-  I'd love if we removed the crosshair cursor; it doesn't seem applicable anymore
- When using https://firefox-dev.tools/debugger-examples/examples/todomvc/app.js as my test file, I add a breakpoint to line 25 by clicking the line number, then press `option` to view the possible column breakpoint locations.  I click the column breakpoint icon and the column breakpoint is added (yay!).  I then click the column breakpoint again to remove it, and the line breakpoint arrow (the blue one that displays on the line number) is removed; we'll want to prevent that.

This is looking awesome.  Great work!

File: src/components/Editor/CallSites.js [hunk 2]
```diff
@@ -142,17 +142,34 @@ class CallSites extends Component {
...
+    const { callSites, breakpoints } = this.props;
+
+    const breakpointLines = new Set();
+    if (breakpoints._root !== undefined) {
```
Comment (loganfsmyth) [inline src/components/Editor/CallSites.js]: Naming conventions in JS generally treat `_` as an indication that something is private. In this case, the specifics of `_root` are internal to the `breakpoints` object and shouldn't be used directly. The `breakpoints` object is an `Immutable.js` `Map` object, so if you peak at the docs, you've for all sorts of options. https://facebook.github.io/immutable-js/docs/#/Map

In this case, I'd probably do

```
const breakpointLines = new Set(breakpoints.toIndexedSeq().map(bp => bp.location.line));
```
which takes all the values and discards the keys, and then converts each value into its line number.

Comment (audreyfang) [reply src/components/Editor/CallSites.js]: Thanks for pointing me in the right direction for this! I'll definitely take a look and clean this up.

**Comment by audreyfang** [comment]
@darkwing In terms of next steps, I wanted a bit of feedback before I did a some more extensive testing. I've definitely noticed some issues like the one you mentioned with removing column breakpoints.

**Pull request merged**